templates/aws/cluster_nodes/terraform/pools/outputs.tf

```diff
@@ -2,26 +2,37 @@ output "fqdn" {
   value = aws_route53_record.aws_route53.fqdn
 }
 
+output "internal_fqdn" {
+  value = var.airgap_setup ? aws_route53_record.aws_route53_internal[0].fqdn : null
+}
+
 output "kube_api_host" {
-  value = aws_instance.server[0].public_ip
+  value = aws_instance.server[0].private_ip
```

Comment:
should this always be the internal IP now, or should it check for the airgap variable as in other outputs?

Reply:
I was having issues with it, and even on a vanilla RKE2 cluster this works. I can have a conditional, but since it was working for both setups I decided to keep it like this.